src/components/TeacherViews/Onboarding/ONboarding.test.tsx

```diff
@@ -1,16 +1,60 @@
 import { screen } from "@testing-library/dom";
+import userEvent from "@testing-library/user-event";
+import { act } from "@testing-library/react";
 
```

Comment:
Is there a reason the test file is called `ONboarding`?

Reply:
Yeah i'm not sure, was like that after merging in main, it wont change back.